### PR TITLE
Ensure connection is closed after CLI commands

### PR DESCRIPTION
**Pull request by seallard**
## Description
This PR adds a teardown function to the CLI which runs after each command has been processed, closing the database session and releasing any resources to the database. This might potentially resolve some of the issues we have experienced with alembic.

### Fixed
- Release db resources once CLI command has been processed

### This [version](https://semver.org/) is a
- [ ] **PATCH** - when you make backwards compatible bug fixes or documentation/instructions

### CODE REVIEW

**Review by diitaz93 — APPROVED** [review]
LGTM 👍

File: cg/cli/base.py [hunk 1]
```diff
@@ -26,11 +27,19 @@
...
 from cg.store.database import create_all_tables, drop_all_tables, get_tables
+from cg.store.database import get_scoped_session_registry
 
 LOG = logging.getLogger(__name__)
 LEVELS = ["DEBUG", "INFO", "WARNING", "ERROR"]
 
 
+def teardown_session():
+    """Ensure that the session is closed and all resources are released to the connection pool."""
+    registry: Optional[scoped_session] = get_scoped_session_registry()
```
Comment (henrikstranneheim) [inline cg/cli/base.py:38]:
```suggestion
    registry: scoped_session | None = get_scoped_session_registry()
```

**Review by henrikstranneheim — APPROVED** [review]
Awesome 👍

**Comment by seallard** [comment]
Deployed and tested in stage

**Pull request merged**